packages/angular_devkit/build_angular/src/utils/process-bundle.ts

```diff
@@ -511,11 +511,11 @@ function createReplacePlugin(replacements: [string, string][]): PluginObj {
 
 const USE_LOCALIZE_PLUGINS = false;
 
-async function createI18nPlugins(
+export async function createI18nPlugins(
   locale: string,
   translation: unknown | undefined,
-  missingTranslation: 'error' | 'warning' | 'ignore',
-  localeDataContent: string | undefined,
+  missingTranslation?: 'error' | 'warning' | 'ignore',
```

Comment:
`missingTranslation: 'error' | 'warning' | 'ignore' = 'ignore',`

And remove all the checks from above.

Reply:
I made it a required param and ensured that a suitable choice is made in all callers.